Tasks/DownloadSecureFile/task.loc.json

```diff
@@ -12,7 +12,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 1,
-    "Minor": 126,
+    "Minor": 127,
```

Comment:
`127` here should also be `132` but the task build process should automatically update this file to match the value you have in `task.json`.